root/static/styles/layout.less

```diff
@@ -147,6 +147,8 @@ pre code {
 .align-left { text-align: left !important; }
 .align-top { vertical-align: top; }
 
+.float-right { float: right; }
```

Comment:
Please let me know if it'd be better to add a dedicated rule in `root/static/styles/edit.less`. I didn't see any obvious place for it there, so I wasn't sure if adding a selector like `#edits .top-submit` (and setting the appropriate class above) would be too gross.

Reply:
See https://github.com/metabrainz/musicbrainz-server/pull/3072#discussion_r1382393723